source/extensions/filters/http/aws_lambda/aws_lambda_filter_config_factory.cc

```diff
@@ -17,21 +17,27 @@ AWSLambdaFilterConfigFactory::createFilterFactoryFromProtoTyped(
         &proto_config,
     const std::string &stats_prefix,
     Server::Configuration::FactoryContext &context) {
+  auto& server_context = context.serverFactoryContext();
 
-
+  // ServerFactoryContext::clusterManager() is not available during server initialization
+  // therefore we need to pass absl::nullopt in lieu of the server_context to prevent
+  // the upstream code from attempting to access the method. https://github.com/envoyproxy/envoy/issues/26653
   auto chain = std::make_unique<Extensions::Common::Aws::DefaultCredentialsProviderChain>(
-          context.api(), Extensions::Common::Aws::Utility::fetchMetadata);
-  auto sts_factory = StsCredentialsProviderFactory::create(context.api(),
-                                            context.clusterManager());
+          server_context.api(), absl::nullopt /* ServerFactoryContextOptRef context */,
+          // We pass an empty string if we don't have a region
```

Comment:
Should we default to `us-east-1`?

Reply:
after some reflection I think it could be very frustrating to debug issues related to default values in this section -- I think I prefer leaving this as-is